explore.html

```diff
@@ -511,6 +511,7 @@ <h2>support</h2>
         </div>
     </footer>
 
+    <script src="https://cdn.jsdelivr.net/npm/underscore@1.12.0/underscore-min.js"></script>

```

Comment:
what is the size impact of this? can we do better?

Reply:
let's add lighthouse scores on each PR

Reply:
1. working on compressing the image, optimizing code, and focusing on the lighthouse report.
2. Added lighthouse to the repo created GitHub actions and status